src/main/java/org/dita/dost/writer/DitaWriterFilter.java

```diff
@@ -147,8 +147,11 @@ private Attributes processAttributes(final String qName, final Attributes atts)
     for (int i = 0; i < attsLen; i++) {
       final String attName = atts.getLocalName(i);
       final String origValue = atts.getValue(i);
+      final String uri = atts.getURI(i);
       String attValue = null;
-      if (origValue.equals(ATTR_VALUE_DITA_USE_CONREF_TARGET)) {
+      if (!uri.isEmpty()) {
```

Comment:
@jelovirt will know better than I do, but is this a reliable indicator of an _unexpected_  namespace? Wondering if there is any impact on the handful of attributes with `ditaarch:` or `xml:` but I don't think this if/else/else currently has any processing for those.